client/src/ui/BitbakeCommands.ts

```diff
@@ -544,6 +545,32 @@ async function openRecipeWorkdirCommand (bitbakeWorkspace: BitbakeWorkspace, bit
   await vscode.commands.executeCommand('vscode.openFolder', recipeWorkdirURI, { forceNewWindow: true })
 }
 
+export let isTaskexpStarted = false
+
+export async function examineDependenciesTaskexp (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjectScanner: BitBakeProjectScanner, uri?: any): Promise<void> {
+  if (isTaskexpStarted) {
```

Comment:
can this happen though?
bitbake driver has a lock that should only allow one process at a time with a few exceptions.

Reply:
In that case, should we really forbid two taskexp? I don't think there's a limitation with spawning it twice.

Reply:
Without it, a new blank terminal opens and does nothing until the first instance of taskexp is closed. I've been thinking it is better to tell the user taskexp is already running, so they know they have to close it first if they want to examine an other recipe.

In fact, the user won't be able to launch any other command while taskexp is running. I wonder if we should tell the user, or having `BitBake Running taskexp...` in the status bar is obvious enough.